nox/virtualenv.py

```diff
@@ -474,18 +479,26 @@ def create(self) -> bool:
 
             return False
 
-        if self.venv_or_virtualenv == "virtualenv":
+        if self.venv_backend == "virtualenv":
             cmd = [sys.executable, "-m", "virtualenv", self.location]
             if self.interpreter:
                 cmd.extend(["-p", self._resolved_interpreter])
+        elif self.venv_backend == "uv":  # pragma: >=3.8 cover
+            cmd = [
+                "uv",
+                "virtualenv",
```

Comment:
```suggestion
                "virtualenv",
                "--seed",
```

See point 3 in https://github.com/wntrblm/nox/pull/762#issuecomment-1951369003

Reply:
IMO, there's no need for pip (or setuptools or wheel for Python <3.11) in the environments - since we have `.install` instead of manually interacting with pip (hurray!), we don't have to guarantee pip is present (especially since this is opt-in; if we change the default to use uv if present, then we could install pip for back-compat purposes, maybe). Someone can install with `session.install("pip")`.

Reply:
(I think the issue in 3. is still solvable, by the way, another way)